client/src/pages/Home.jsx

```diff
@@ -22,77 +26,140 @@ function Home() {
       });
   }, []);
 
+  // Recipe King Modals
+  const editMode = true;
+  const [isLoggedIn, setIsLoggedIn] = useState(false); // Here
```

Comment:
kéne egy context provider az app jsx-ben ami az egész appban elérhetővé teszi ez a statet- hogy a user logged-in vagy nem